site.hs

```diff
@@ -25,11 +25,10 @@ main = hakyll $ do
 
     match "index.html" $ do
         route idRoute
-        compile $ do
-            getResourceBody
-                >>= applyAsTemplate defaultContext
-                >>= loadAndApplyTemplate "templates/boilerplate.html" defaultContext
-                >>= relativizeUrls
+        compile $ getResourceBody
+            >>= applyAsTemplate defaultContext
+            >>= loadAndApplyTemplate "templates/boilerplate.html" defaultContext
+            >>= relativizeUrls
 
     match "affiliates/*.markdown" $ do
         route $ setExtension "html"
```

Comment:
@myShoggoth I think we can remove this routing to affiliates pages, I don't see any access currently on prod